content/en/blog/gsoc_2023/week1.md

```diff
@@ -0,0 +1,48 @@
+
+
+# New beginings 🚀
+
+By contributing to the project prior to the GSoC period, I had the opportunity to understand the project's ecosystem, gain insights into its development processes, and establish myself as a valuable member of the community. It provided me with a deeper understanding of the project's goals and challenges, enabling me to better align my proposed solutions and goals for the GSoC period.
+
+During this pre-GSoC contribution phase, [Jason Dellaluce](https://github.com/jasondellaluce), my mentor, played a crucial role in guiding and supporting me. Jason provided valuable feedback on my early contributions, helped me navigate the project's codebase, and encouraged me to explore new areas for improvement. This continued collaborative relationship with Jason contributed to my growth as a contributor and helped set the stage for a successful GSoC journey.
+
+As Week 1 of the official GSoC period began, my pre-GSoC contributions served as a foundation for diving deeper into the project. The knowledge and familiarity I had gained during this time allowed me to hit the ground running and make meaningful progress from the very beginning. It also reinforced the strong mentor-mentee relationship with Jason, as we had already established a rapport and had a shared understanding of the project's context.
+
+Here is a summary of some of the pull requests (PRs) I have submitted up until today.
+
+- [PR #1: Test For command falco -i (ignore default events)](https://github.com/falcosecurity/testing/pull/8)
+    - Description: The [falcosecurity/testing](https://github.com/falcosecurity/testing) repository contains regression test suite for Falco and other tools in its ecosystem. I contributed by adding a new test case for falco's help output when used with -i flag. I contributed to two more of such falgs. 
+
+- [PR #2: Semver check for RequiredAPIVersion values](https://github.com/falcosecurity/plugin-sdk-go/pull/73)
+    - Description: This pr checks if user's RequiredAPIVersion follows semver system and checks if is compatible with internally-supported API version
```

Comment:
I would user "semantic versioning" instead of "semver" to make it clearer.
```suggestion
    - Description: This PR checks if the user's RequiredAPIVersion follows the semver system and checks if it is compatible with the internally-supported API version.
```